invenio.cfg

```diff
@@ -411,24 +410,29 @@ RDM_FILES_DEFAULT_MAX_FILE_SIZE = 50 * 10 ** 9  # 50GB
 
 JOBS_ADMINISTRATION_ENABLED = True
 
-RDM_RECORDS_IDENTIFIERS_SCHEMES = {**RDM_RECORDS_IDENTIFIERS_SCHEMES,
-                                   **{"cds_ref": {"label": _("CDS Reference"),
-                                                  "validator": always_valid,
-                                                  "datacite": "CDS"},
-                                      "aleph": {"label": _("Aleph number"),
-                                                "validator": schemes.is_aleph,
-                                                "datacite": "ALEPH"},
+RDM_RECORDS_IDENTIFIERS_SCHEMES ={
+                                "cdsrn": {"label": _("CDS Reference"),
+                                                "validator": always_valid,
+                                                "datacite": "CDS"},
+                                "aleph": {"label": _("Aleph number"),
+                                            "validator": schemes.is_aleph,
+                                            "datacite": "ALEPH"},
+                                "lcds": {"label": _("CDS"),
```

Comment:
This one should be `cds` only. Let's discuss IRL because there is the same issue in another PR.
See: https://github.com/CERNDocumentServer/cds-rdm/issues/532